app/src/main/res/layout/activity_register.xml

```diff
@@ -0,0 +1,168 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="#FDE54E">
+
+    <ImageView
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:src="@drawable/deuvox_logo_black"
+        android:layout_marginTop="25dp"
+        android:layout_marginStart="24dp"/>
+
+    <ImageView
+        android:layout_width="411dp"
+        android:layout_height="520dp"
+        android:layout_alignParentStart="true"
+        android:layout_alignParentTop="true"
+        android:layout_marginTop="256dp"
+        android:background="@drawable/curved_header" />
```

Comment:
is it responsive? please try in layout validation

i think you can try wrapping it in a container, and apply a background to the container